states/huntsman/housekeeping.py

```diff
@@ -1,5 +1,17 @@
+def deactivate_camera_cooling(pocs):
+    """
+    Deactivate camera cooling before entering sleeping state.
+    """
+    pocs.logger.debug('Deactivating camera cooling before entering sleeping state.')
+    for cam in pocs.observatory.cameras.values():
+        if cam.is_cooled_camera:
+            cam.cooling_enabled = False
```

Comment:
I'm not sure whether `cooling_enabled` is handled like that - I can't see it in `POCS`. Maybe @AnthonyHorton can clarify? 

Maybe there isn't much need to have it as a separate function, but I like the idea of keeping the state codes as abstract as possible. 

Not sure moving the function to `observatory` is really necessary at the moment given its only going to be called here, unless cooling is turned off somewhere else?

Reply:
On second thought, maybe it would be better to add `deactivate_camera_cooling` and `activate_camera_cooling` methods to `HuntsmanObservatory`.

Reply:
Currently if you try to set `cooling_enabled` on a camera that doesn't implement that control it will raise a `NotImplementedError`. So, either need to check `is_cooled_camera` before setting it, or wrap the setting in a `try ... except NotImplementError` or, perhaps best of all, wrap it in `with suppress(NotImplementedError):`.

@danjampro this is all in the `pocs.camera.camera.AbstractCamera` base class.